### DISCUSSION TITLE
How well does Splink work on structured data?

### DISCUSSION THREAD

**Discussion by kodeplex** [moj-analytical-services/splink#1533] [General]
How well does it work for quite structured data? I have the following use case and I am doubting between Python Record Linkage Toolkit, Dedupe and Splink for accuracy. For Customer database:

-Matching Columns:
a) Name: With some variations in writing.
b) Postal Code
c) VAT

3) **Address (Lot of variations here),** because address aren't always written in same shape/form.

**Comment by RobinL** [comment]
Yes, that's the sort of data Splink is designed to handle

**Reply by kodeplex** [reply]
Thank you!!
